### PR TITLE
Fix issue calculating the stripIndex for planar tiff's

### PR DESCRIPTION
**Pull request by brianpopow**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/SixLabors/ImageSharp/pulls) open
- [x] I have verified that I am following the existing coding patterns and practice as demonstrated in the repository. These follow strict Stylecop rules :cop:.
- [x] I have provided test coverage for my change (where applicable)

### Description

Fixes a bug calculating the stripIndex for planar tiff images.

example strip offsets and byte counts from `flower-rgb-planar-08-6strips.tiff` (tiffinfo -s)

```
6 Strips:
            offsets, byte count
      0: [       8,     2190]
      1: [    2198,      949]
      2: [    3147,     2190]
      3: [    5337,      949]
      4: [    6286,     2190]
      5: [    8476,      949]
```

@IldarKhayrutdinov could you please review this change when you have time?

### CODE REVIEW

**Comment by IldarKhayrutdinov** [comment]
@brianpopow yeah, it seems there was a bug, now looks good!

**Comment by brianpopow** [comment]
> yeah, it seems there was a bug, now looks good!

@IldarKhayrutdinov ok, thanks for taking a look!

File: tests/ImageSharp.Tests/Formats/Tiff/TiffDecoderTests.cs [hunk 1]
```diff
@@ -84,6 +84,12 @@ public void ByteOrder(string imagePath, ByteOrder expectedByteOrder)
         public void TiffDecoder_CanDecode_Uncompressed<TPixel>(TestImageProvider<TPixel> provider)
             where TPixel : unmanaged, IPixel<TPixel> => TestTiffDecoder(provider);
 
+        [Theory]
+        [WithFile(FlowerRgb888Planar6Strips, PixelTypes.Rgba32)]
+        [WithFile(FlowerRgb888Planar15Strips, PixelTypes.Rgba32)]
+        public void TiffDecoder_Planar<TPixel>(TestImageProvider<TPixel> provider)
```
Comment (IldarKhayrutdinov) [inline tests/ImageSharp.Tests/Formats/Tiff/TiffDecoderTests.cs:90]: new test files are good,
we needs to add more test files for tiff

**Review by IldarKhayrutdinov — APPROVED** [review]

File: tests/ImageSharp.Tests/Formats/Tiff/TiffDecoderTests.cs (hunk 1 above)
Comment (brianpopow) [reply tests/ImageSharp.Tests/Formats/Tiff/TiffDecoderTests.cs:90]: we had test files for planar, but only with 3 strips, so the bug was not triggered.

**Pull request merged**